lib/Documentable/DocPage/Factory.pm6

```diff
@@ -1,6 +1,5 @@
 
 use Documentable;
-use Pod::Load;
 use Documentable::Config;
```

Comment:
Did you check if it's no longer used to eliminate it from dependencies?

Reply:
It's still used in the tests.